content/docs/capabilities/getting-users-identity.md

```diff
@@ -21,70 +34,112 @@ import ReactApp from '../../examples/js-sdk/react-app.md';
   allowfullscreen></iframe>
 
-This article describes how to retrieve a user's identity from a Pomerium-managed application. Pomerium uses JSON web tokens (**[JWT]**) to attest that a given request was handled by Pomerium's authorization service.
+## Overview
 
-## Prerequisites
+JWTs provide a secure and efficient means to authenticate and authorize users before they can access upstream services behind Pomerium. When configured for JWT authentication, Pomerium sends its own JWT to the upstream service. By verifying the Pomerium JWT, the upstream service can:
 
-To secure your app with signed headers, you'll need the following:
+- Confirm that the Pomerium Proxy service handled the client request before forwarding it. (This capability offers an easy alternative to [client-side mTLS](/docs/capabilities/mtls-clients), which can be difficult to configure depending on your infrastructure or environment.)
+- Make application-level authorization decisions based on the user's associated identity information.
 
-- An application you want users to connect to.
-- A [JWT library] with support for the `ES256` signing algorithm.
+## Why JWT authentication?
 
-## JWT data
+### Identity verification
 
-When the [Pass Identity Headers] route option is enabled, the user's associated identity information will be included in a signed attestation JWT. This JWT is added to each upstream request in the header `X-Pomerium-Jwt-Assertion`. The signed attestation JWT is also available at the special `/.pomerium/jwt` endpoint of any URL handled by Pomerium.
+JWT authentication through Pomerium enables an upstream service to verify a user's identity based on claims contained in the JWT. Pomerium signs and issues a new JWT based on the ID token received from the service's configured identity provider, so the upstream service can verify that the claims were not modified in transit.
 
-The JWT will contain at least the following claims:
+### Request verification
 
-| claim | description |
-| :-: | --- |
-| `exp` | Expiration time in seconds since the UNIX epoch. |
-| `iat` | Issued-at time in seconds since the UNIX epoch. |
-| `aud` | The domain for the upstream application (e.g. `httpbin.corp.example.com`). |
-| `iss` | Same as the `aud` claim. |
-| `sub` | The user's ID, as specified by the identity provider. |
-| `email` | The user's email address. |
-| `groups` | The user's group memberships (if supported for the identity provider). |
-| `name` | The user's full name, as specified by the identity provider. |
+Pomerium places the newly minted JWT in a [JWT assertion header](#jwt-assertion-header). The upstream service should only accept the incoming request if it satisfies all [JWT validation](#jwt-validation) conditions.
+
+By validating the JWT, the upstream service can assert that:
+
+- The request originated from Pomerium
+- The user was authenticated
+- The request was authorized in accordance with the route's authorization policy
+
```

Comment:
let's add a note that JWT assertion currently does not carry any of the path information, in case there are i.e. multiple routes configured for the same upstream host with different paths - imagine a dedicated route that allows a limited set of users to reach `/admin`. an upstream application currently cannot assert which Pomerium route the JWT corresponds to.